osi_trafficsign.proto

```diff
@@ -468,7 +468,7 @@ message TrafficSign
                 //
                 TYPE_RIGHT_BEFORE_LEFT_NEXT_INTERSECTION = 3;
```

Comment:
In the following defines, DANGER is missing again.
I'd propose to handle this consistently and thuss remove  DANGER from all types.